proposals/csharp-14.0/field-keyword.md

```diff
@@ -520,6 +528,12 @@ One reason we shied away from using nullability attributes here is that the ones
 - If we pursued this approach, we would consider adding a warning when `[field: AllowNull]` is used, suggesting to also add `MaybeNull`. This is because AllowNull by itself doesn't do what users need out of a nullable variable: it assumes the field is initially not-null when we never saw anything write to it yet.
 - We could also consider adjusting the behavior of `[field: MaybeNull]` on the `field` keyword, or even fields in general, to allow nulls to also be written to the variable, as if `AllowNull` were implicitly also present.
 
+#### Infer the initial flow state of the `field`
+
+Instead of defining the behavior in terms of the `field`'s nullable annotation, we could just the initial flow state of the `field` in accessors. This would remove the surprising behavior that the inferred nullable annotation is not exposed in the symbol APIs. Also, it would reflect the reality that it's ok to assign a possible null value to the `field`--the other forms of analysis we have, would succeed in identifying any resulting null safety issues.
```

Comment:
```suggestion
Instead of defining the behavior in terms of the `field`'s nullable annotation, we could just define it in terms of the initial flow state of the `field` in accessors. This would remove the surprising behavior that the inferred nullable annotation is not exposed in the symbol APIs. Also, it would reflect the reality that it's ok to assign a possible null value to the `field`--the other forms of analysis we have, would succeed in identifying any resulting null safety issues.
```